patterns/1-initial/secure-innersource.md

```diff
@@ -0,0 +1,160 @@
+# Title
+
+Secure InnerSource
+
+# Patlet
+
+While InnerSource flourishes in environments with a high degree of shared code, Security/Legal prefers the limitation of source code access to only those that need it.
+By making Security/Legal part of the team, introducing explicit sharing levels, and defining what qualifies as sensitive information, code sharing can be facilitated while minimizing the associated risks.
```

Comment:
```suggestion
By making Security/Legal part of the team, introducing explicit sharing levels and security policies for shared repositories, as well as defining what qualifies as sensitive information, code sharing can be facilitated while minimizing the associated risks.
```

I have worked in your suggestion.

The Patlet is always a trade-off between repeating the whole solution and staying concise. I can certainly see that adding the info about the "security policies" will help the readers though.